src/components/BookAppointment.jsx

```diff
@@ -216,41 +27,116 @@ const TwoLastSteps = (props) => {
 };
 
 function BookAppointment({ t }) {
+    const [currentStep, setCurrentStep] = useState(0);
+
+    const initialValues = {
+        dateTime: new Date(),
+        counselingType: "individual",
+        relationshipStatus: "single",
+        therapyBefore: "yes",
+        specificQualities: "maleCounselor",
+        issue: "depression",
+        bringsHere: "",
+    };
+
+    const [values, setValues] = useState(initialValues);
+
+    const handelChange = ({ target }) => {
+        setValues({ ...values, [target.name]: target.value });
+    };
+
+    const onSubmit = async () => {
+        axios
+            .post("/api/appointments", values, {
+                headers: { "Content-Type": "application/json" },
+            })
+            .then((res) => {
+                if (res.data.success === 0) {
+                    setCurrentStep(currentStep + 1);
+                }
+            });
```

Comment:
Since this snippet of code will be repeated many times throught project, make it as utils and pass needed props.